package.json

```diff
@@ -438,17 +438,22 @@
         },
         "puppet.installDirectory": {
           "type": "string",
-          "default": null,
-          "description": "The fully qualified path to the Puppet install directory. This can be a PDK or Puppet Agent installation. For example: 'C:\\Program Files\\Puppet Labs\\Puppet' or '/opt/puppetlabs/puppet'. If this is not set the extension will attempt to detect the installation directory"
+          "default": "",
```

Comment:
Does this need a default?

Reply:
The `null` setting was causing a big red error when you looked at the settings in the UI, so I set it to an empty string. Testing without `default` set shows no error and allows input so I'll remove the `default` key.